x-pack/plugin/esql/qa/testFixtures/src/main/resources/union_types.csv-spec

```diff
@@ -514,6 +514,21 @@ mc:l | count:l
 7    | 2
 ;
 
+multiIndexTsLongStatsStats
```

Comment:
This is not the test that tests the fix in the PR; but is a useful test that was used to flush out some related behaviors, so I wanted to keep it. For the test that really matters, look up at line 284, which uses the capability `union_types_inline_fix`, and was already merged as a result of the previous fix, but had the wrong capability name, and so was ignored and we missed the failure. This PR adds that capability name, enabling this test, and fixes the test.